CONTRIBUTING.rst

```diff
@@ -83,9 +83,52 @@ Using a custom ``virtualenv``
     $ cd ${HOME}/hack-on-google-cloud-python
     $ ${VENV}/bin/python setup.py install
 
+- To keep your ``virtualenv`` from getting checked-in, you can make a
+  global gitignore file::
+
+  $ echo "${ENV_NAME}" >> ~/gitignore_global
+  $ git config --global core.excludesfile ~/.gitignore_global
```

Comment:
> make the contributing path as happy as possible

That's fine, but we don't need to also teach the contributors how to be Python devs. The more non-project related content we have in this doc, the less useful it is to readers.